server/api/auth_handler.py

```diff
@@ -18,8 +18,24 @@ def decorated_function(*args, **kwargs):
                 auth_token = User.decode_auth_token(auth_token)
                 if not isinstance(auth_token, str):
                     g.user = User.query.filter_by(id=auth_token).first()
+                else:
+                    responseObject = {
+                        "status": "fail",
+                        "message": "User is not logged in."
+                    }
+                    return make_response(jsonify(responseObject)), 401
+            else:
+                responseObject = {
+                    "status": "fail",
+                    "message": "User is not logged in."
+                }
+                return make_response(jsonify(responseObject)), 401
```

Comment:
To avoid repetition, you can raise an exception instead, which will be caught below.

Reply:
I fixed the issue by raising a generic exception, is that okay or do I need to raise something more specific?

Reply:
Good for now 👍